### PR TITLE
Added Vietnamese translation

### PR DESCRIPTION
**Pull request by QuanMCPC**
Just a small contribution :)

### CODE REVIEW

**Review by BachLe2000 — APPROVED** [review]
Approved, hope help the code owner!

File: vi/index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (tulir) [inline vi/index.html:2]:
```suggestion
<html lang="vi">
```

**Review by tulir — APPROVED** [review]

**Comment by QuanMCPC** [comment]
So will the Vietnamese version be merged or is these something that need to be resolved?

**Comment by mtatko** [comment]
This wasn't merged for some reason, however I see tulir has approved it, so I'm gonna go ahead and merge it.

**Pull request merged**